ThalesGroup/security-risk-assessment-tool
# Bugfix/valid risks with empty fields

Added conditions for validating SupportingAsset and BusinessAsset before setting the select input values in Risk description section
 
Added conditions for validating Vulnerabilities in Attack Paths before setting the select inputs in Risk evaluation section 

Added error handling and messages for Risk evaluation section when Saving a file

Steps to validate SupportingAsset & BusinessAsset:
Fill the required in the Risk description section of a risk
Delete the chosen BusinessAsset
Go back to the Risk Panel
See the risk is now invalid (Error when saving) and red
See the select value of BusinessAsset & SupportingAsset as "Select..."

Steps to validate Vulnerabilities in Attack Paths :
Add a Vulnerability in Attack Paths
Delete the chosen Vulnerability 
Go back to the Risk Panel
See the risk is now invalid (Error when saving) and red
